content/v2.0/account-management/pricing-plans.md

```diff
@@ -69,3 +70,9 @@ To protect against any intentional or unintentional harm, Usage-based Plans incl
   - Unlimited PagerDuty, Slack, and HTTP notification endpoints
 
 _To request higher rate limits, contact [InfluxData Support](mailto:support@influxdata.com)._
+
+{{% cloud %}}
```

Comment:
No need for the cloud shortcode here. This entire page is cloud-only.

Reply:
https://github.com/influxdata/docs-v2/pull/1257#discussion_r463290837

Reply:
Since it's a non-standard plan/option, I thought it made sense to set this off visually.